unreleased.md

```diff
@@ -1,13 +1,47 @@
 # Unreleased
 
+## .NET Upgrade
+
+Baked now supports .NET 9! Below you can find a task list to upgrade your
+projects.
+
+```markdown
+- [ ] Upgrade .NET and C# versions
+  - [ ] in projects
+  - [ ] in docker files
+  - [ ] in GitHub workflows
+- [ ] Upgrade Baked version
+- [ ] You can use `GeneratedRegex`es in properties instead of methods
+- [ ] If `Base64` encoded information is carried in the url, use `Base64Url`.
+  - [ ] If need to take parameters with an `Array` using params
+  and then convert to `IEnumerable` type, use `IEnumerable` instead of `Array`.
```

Comment:
this one might need to be more direct;

```markdown
- [ ] `params` arguments should be converted from arrays to `IEnumerable`
```

Reply:
Do you really need this directive? I said it to say that if the array is cast to list, readonlyspan or something like that in the rest of the code, “you don't need this, you can get directly IEnumareble”.

Reply:
Okay, now I realize it's the same thing.